results.go

```diff
@@ -129,6 +129,16 @@ type Link struct {
 	Rel  string `mapstructure:"rel"`
 }
 
+// PrettyPrintJSON creates a string containing the links as
+// pretty-printed JSON. It is useful for displaying the results.
+func (link Link) PrettyPrintJSON() string {
```

Comment:
What is the use-case for this? The `gophercloud.Link`s should be formatted if they are in the result and `gophercloud.Result.PrettyPrintJSON` is called. When would you need to format the links separately?